### DISCUSSION TITLE
Add copy output of ray

### DISCUSSION THREAD

**Discussion by viezel** [spatie/ray#127] [Ideas]
I cannot copy the output of logged data in Ray, I cannot easily copy it in the right format.

e.g. the following is not a valid array. 

```php
  "instructions" => "blah blah ▶"
  "ingredients" => array:6 [▼
    0 => array:4 [▶]
    1 => array:4 [▼
      "info" => "smeltet"
      "unit_id" => 3
      "quantity" => 40
      "ingredient_id" => "380"

```

I would love a button to copy the output correctly, and not the displayed version

**Comment by AdrianMrn** [comment]
Good idea, we'll discuss this internally.
